packages/terra-form-select/CHANGELOG.md

```diff
@@ -5,7 +5,10 @@
 ## 6.51.0 - (November 21, 2023)
 
 * Added
-  * Added 'aria-invalid' attribute which will be set to true for error input fields and false when resolving errors. 
+  * Added 'aria-invalid' attribute which will be set to true for error input fields and false when resolving errors.
```

Comment:
```suggestion
  * Added 'aria-invalid' attribute for `terra-form-select-combobox`
```

Reply:
Duplicate entry and already 'aria-invalid' attribute changes has been merged with main branch.